fbw-a380x/src/wasm/systems/a380_systems_wasm/src/spoilers.rs

```diff
@@ -35,5 +35,111 @@ pub(super) fn spoilers(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Er
         Variable::named("HYD_SPOILERS_RIGHT_DEFLECTION"),
     );
 
+    // FIX ME: Remove this section once spoiler animation is back to linear in the model
+    builder.map(
```

Comment:
Shouldn't these be `ExecuteOn::PostTick`? Otherwise you always use the values from the previous frame if I understand the aspect layer correctly.